lib/carbon/client.py

```diff
@@ -2,8 +2,9 @@
 from time import time
 from six import with_metaclass
 
+from OpenSSL import SSL
```

Comment:
Will it make pyOpenSSL required dependency then? Could it be optional then?

Reply:
I've updated to make it optional.